app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizardActions.js

```diff
@@ -22,10 +22,14 @@ export const mappingWizardExitedAction = () => dispatch => {
   });
 
   // Dispatch reset for all the wizard step forms here
-  dispatch(reset('mappingWizardGeneralStep'));
-  dispatch(reset('mappingWizardClustersStep'));
-  dispatch(reset('mappingWizardDatastoresStep'));
-  dispatch(reset('mappingWizardNetworksStep'));
+  dispatch(
+    destroy(
```

Comment:
is `destroy` better than `reset`?

If so, should we also make this change in the Plan Wizard?

Reply:
I changed to `destroy` because `reset` reverts the form state to its `initialValues`.  Now that we're using the initial values to pre-set the forms when editing, `reset` doesn't behave as expected in edit mode (perhaps we are abusing redux-form's initial value functionality here)

`destroy` completely wipes out the form's state, so the downside is that it adds one more step to the process.  Redux-form needs to reinitialize the forms when re-entering the steps

Reply:
I see, Ok.